src/main/java/org/embulk/util/guess/timeformat/GuessMatch.java

```diff
@@ -192,7 +194,13 @@ public void mergeFrom(final TimeFormatMatch anotherInGroup) {
         final List<GuessOption> anotherPartOptions = another.getPartOptions();
 
         for (int i = 0; i < this.partOptions.size(); i++) {
-            if (this.partOptions.get(i) == null) {
+            final GuessOption option = this.partOptions.get(i);
+            if (option == null || option == GuessOption.NIL) {
```

Comment:
`partOptions` has `NIL` instead of `null`. It should have been used.

https://github.com/embulk/embulk-util-guess/blob/v0.1.3/src/main/java/org/embulk/util/guess/timeformat/GuessOption.java